main.tf

```diff
@@ -1,227 +1,54 @@
-
-  billing_mode = var.ddb_billing_mode
-
-  autoscale_read_schedule  = var.ddb_autoscale_read_schedule
-  autoscale_write_schedule = var.ddb_autoscale_write_schedule
-
-  hash_key = "key"
-
-  dynamodb_attributes = [{
-    name = "key"
-    type = "S"
-  }]
-
-  ttl_enabled = false
-
-  tags = {
-    Model = "kvstore_${try(module.this.attributes[0], "")}"
-  }
-}
-
-module "task_label" {
-  source      = "cloudposse/label/null"
-  version     = "0.25.0"
-  attributes  = ["task"]
-  label_order = var.label_orders.iam
+  label_value_case = "none"
+  label_order      = var.label_orders.kvstore
+  tenant           = "kvstore"
 
   context = module.this.context
 }
 
-data "aws_iam_policy_document" "ecs_task" {
-  statement {
-    effect  = "Allow"
-    actions = ["sts:AssumeRole"]
-
-    principals {
-      type        = "Service"
-      identifiers = ["ecs-tasks.amazonaws.com"]
-    }
-  }
-}
-
-resource "aws_iam_role" "ecs_task" {
-  count = var.redis_enabled ? 1 : 0
-
-  name               = module.task_label.id
-  assume_role_policy = data.aws_iam_policy_document.ecs_task.json
-  tags               = module.task_label.tags
-}
-
-module "exec_label" {
-  source      = "cloudposse/label/null"
-  version     = "0.25.0"
-  attributes  = ["exec"]
-  label_order = var.label_orders.iam
-
-  context = module.this.context
-}
-
-data "aws_iam_policy_document" "ecs_task_exec" {
-  statement {
-    actions = ["sts:AssumeRole"]
-
-    principals {
-      type        = "Service"
-      identifiers = ["ecs-tasks.amazonaws.com"]
-    }
-  }
-}
-
-resource "aws_iam_role" "ecs_exec" {
-  count = var.redis_enabled ? 1 : 0
-
-  name               = module.exec_label.id
-  assume_role_policy = data.aws_iam_policy_document.ecs_task_exec.json
-  tags               = module.exec_label.tags
-}
-
-data "aws_iam_policy_document" "ecs_exec" {
-  statement {
-    effect    = "Allow"
-    resources = ["*"]
-
-    actions = [
-      "ssm:GetParameters",
-      "ecr:GetAuthorizationToken",
-      "ecr:BatchCheckLayerAvailability",
-      "ecr:GetDownloadUrlForLayer",
-      "ecr:BatchGetImage",
-      "logs:CreateLogGroup",
-      "logs:CreateLogStream",
-      "logs:PutLogEvents"
-    ]
-  }
-}
-
-resource "aws_iam_role_policy" "ecs_exec" {
-  count = var.redis_enabled ? 1 : 0
-
-  name   = module.exec_label.id
-  policy = data.aws_iam_policy_document.ecs_exec.json
-  role   = aws_iam_role.ecs_exec[0].id
-}
-
-module "container_definition" {
-  count   = var.redis_enabled ? 1 : 0
-  source  = "cloudposse/ecs-container-definition/aws"
-  version = "0.58.1"
-
-  container_image              = "${var.redis_image_repository}:${var.redis_image_tag}"
-  container_name               = local.redis_container_name
-  container_cpu                = var.redis_cpu_size
-  container_memory_reservation = var.redis_service_memory_size
-
-  port_mappings = [
+module "ddb" {
+  count   = var.ddb_enabled ? 1 : 0
+  source  = "terraform-aws-modules/dynamodb-table/aws"
```

Comment:
why are you switching away from our own module to this one?

Reply:
In this case for consistency as I don't want to use multiple different ddb modules.